server/server.js

```diff
@@ -0,0 +1,16 @@
+import express from 'express';
+import webpack from 'webpack';
+import webpackDevMiddleware from 'webpack-dev-middleware';
```

Comment:
All this middleware needs to be paramaterized by development env. Don't want to use this in prod and the way this file is written means we have to have these deps in prod.

Reply:
I was attempting to only import the middleware if the environment was development, but it would throw errors. Could you clarify how to set up the server differently for development and production environments please? Thanks for your help!

Reply:
Make a middleware that checks for env and uses webpack middleware or write a function that checks env and then uses webpack if dev

Reply:
Okay, thanks for the clarification! :pray: